src/lib/parquet/src/Flow/Parquet/Writer.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Flow\Parquet;
 
+use Composer\InstalledVersions;
```

Comment:
Adding this class means that `composer.json` needs to have a clear dependency on `composer-runtime-api:^2.0` both here and in the main folder.

Reply:
Could you elaborate? I thought this comes with the composer itself and it's always available through autoload. What exactly is the link to that package you mentioned on packagist?

Reply:
It's true, but from the perspective of dependencies, this class doesn't exist till you use Composer. It's not a real package, it's a virtual one but allows marking it as a clear dependency, similar fix here: https://github.com/elastic/elastic-transport-php/issues/8#issuecomment-1211937261